src/CRFTDRegistry.sol

```diff
@@ -27,16 +27,42 @@ error ImplementationNotApproved();
 /// @notice CRFTD proxy registry
 contract CRFTDRegistry is Owned(msg.sender) {
     event Registered(address indexed user, bytes32 id, uint256 fee);
+    event CollectionRegistered(address indexed user, bytes32 id, uint256 fee);
     event ProxyDeployed(address indexed owner, address indexed proxy);
+    event VIPStatusChanged(address indexed user, bool status);
```

Comment:
I've thought a little bit more and let's actually not include any VIPStatus just yet. I think we could probably handle that directly from our backend instead of on the smart contract.